src/device/udp_unix.rs

```diff
@@ -65,7 +65,7 @@ impl UDPSocket {
                 #[cfg(not(target_os = "linux"))]
                 SO_REUSEPORT,
                 &1u32 as *const u32 as *const c_void,
-                std::mem::size_of::<u32>() as u32,
```

Comment:
just to be safe it is best to use `as _` instead of `as socklen_t`

Reply:
@vkrasnov ?